apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html

```diff
@@ -129,8 +132,9 @@ <h4 class="modal-title bg pull-left">Adjust Payment</h4>
                         </div>
                     </ng-container>
                     <span class="text-light" *ngIf="isUpdateMode">
-                Excess amount: {{invoiceFormDetails.accountDetails.currencySymbol}}{{adjustPayment.totalAdjustedAmount- adjustPayment.grandTotal - depositAmount  | giddhCurrency}}
-            </span>
+                        Excess amount:
```

Comment:
Avoid expression calculation in HTML as it is run in every change detection even if it is not required. Assign this expression to a variable and update that variable when needed

Reply:
This all variable have different value in different action no need to place or assign in single variable because all these is calculating in different way and actions